### ISSUE TITLE
`suffix` path style that would maintain image size

### ISSUE THREAD

**Issue by Clasyc** [cshum/imagor#307]
Now if we set path style to `suffix`, `166x169/top/foobar.jpg` becomes `foobar.45d8ebb31bd4ed80c26e.jpg`

It would be great to have option on maintaining resolution, for example:
`166x169/top/foobar.jpg` becomes `166x169/foobar.45d8ebb31bd4ed80c26e.jpg`
or
`166x169/top/foobar.jpg` becomes `foobar.45d8ebb31bd4ed80c26e_166x169.jpg` 
or any other strategy that you suggest.

And it would be really convenient to receive new generated name in the response header.
For example `"imagor-result-storage-path: foobar.45d8ebb31bd4ed80c26e_166x169.jpg"`

**Comment by cshum** [comment]
not a good idea to modify existing conventions. Instead added added a new one:
```
IMAGOR_RESULT_STORAGE_PATH_STYLE=size
```
You may test it under the master branch docker build: 
https://github.com/cshum/imagor/pkgs/container/imagor/67912489?tag=master
```
ghcr.io/cshum/imagor@sha256:7c362143a2961e7e86604e55ed0cf12dcd3021fd549a63abf541bd2276162e3c
```

**Comment by Clasyc** [comment]
Thank you! Yea my primary presumption was to add another one, instead of modifying existing one.

**Comment by cshum** [comment]
Available in v1.4.0
